src/runtime/crt/host/CMakeLists.txt.template

```diff
@@ -0,0 +1,51 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+# SPDX-License-Identifier: Apache-2.0
+
+cmake_minimum_required(VERSION 3.18)
+set(CMAKE_CXX_STANDARD 11)
+
+project(crt_autogenerated_project C CXX)
+add_executable(main)
+
+set(CRT_LIB_BASE crt/src/runtime/crt)
+set(CRT_LIBS microtvm_rpc_server microtvm_rpc_common aot_executor_module aot_executor graph_executor_module graph_executor common memory)
```

Comment:
I would put one these targets (libs) per line to make it more readable

Reply:
done